database/scripts/generate_report.rb

```diff
@@ -51,3 +50,4 @@ def generate_report(report_name, exclude_set)
 end
 
 generate_report(options[:report_name], options[:exclude])
+puts report_name
```

Comment:
Why is this needed?

Reply:
The default report name is the current DateTime. Because of this, the report name varies every minute. 

The idea is to keep track of which report was generated.